lib/OpenQA/Command.pm

```diff
@@ -11,6 +11,7 @@ use Mojo::Util qw(decode getopt);
 use Mojo::URL;
 use Mojo::File qw(path);
 use Term::ANSIColor qw(colored);
+use open qw(:std :encoding(UTF-8));
```

Comment:
This will probably cause unintended side effects. In my experience it's better to keep STDIN/STDOUT in binary mode and manually encode/decode when necessary on a case by case basis. Even just printing JSON to STDOUT would now require you to decode it before printing to avoid double encoding issues.